docs/reference/migration/migrate_8_0/settings.asciidoc

```diff
@@ -280,3 +280,18 @@ per index data path settings.
 
 *Impact* +
 Discontinue use of the deprecated settings.
+
+[[single-data-node-watermark-setting]]
+.Single data node watermark setting deprecated
```

Comment:
Should we also consider the change in allocation behaviour on a single node to be a breaking change, and indicate that folks should set `cluster.routing.allocation.disk.threshold_enabled: false` if they want to keep the old behaviour?